_includes/contact_form.html

```diff
@@ -353,187 +348,199 @@
   document.addEventListener('DOMContentLoaded', doNotAnswer);
 </script>
 
-<form
-  id="contact-us-form"
-  action="{{ site.contact_form_action }}"
-  method="POST"
-  onSubmit="return remapvalues(this); return true;"
-  class="usa-form"
->
-  <input type="hidden" name="retURL" value="https://login.gov{{ '/' | locale_url }}" />
-  <input type="hidden" name="orgid" value="{{ site.contact_form_orgid }}" />
-  <input type="hidden" name="status" id="status" value="New" />
-  <input type="hidden" id="external" name="external" value="1" />
-  <input type="hidden" name="recordType" id="recordType" value="012t00000004JFu" />
-  {% if site.contact_form_captcha_enabled != true %}
-  <input type="hidden" name="debug" value="1" />
-  {% endif %}
-
-  <p>
-    {{ site.data.[page.lang].settings.contact_page.support_request_form.help_text }}
-  </p>
-
-  <!-- START: Topic -->
-  <label class="usa-label" for="00NU0000004z903"
-    >{{ site.data.[page.lang].settings.contact_page.support_request_form.labels.topic }}</label
+<div>
```

Comment:
Is this new plain `<div>` wrapper necessary for anything?

Reply:
It was being used to add a divider in the original design, and there was an idea at one point to link directly to the form from the nav. I left it in so it'd be easy to add those two things back in the future. Happy to remove if we generally hate extra divs. :-)